python/style.rst

```diff
@@ -545,6 +545,25 @@ methods, ``cls`` should be used instead.
     This is consistent with the naming conventions in PEP 8 as indicated explicitly
     by `upstream <https://mail.python.org/pipermail/python-dev/2018-January/151986.html>`_.
 
+.. _style-guide-py-naming-import:
+
+Import statements SHOULD follow a standard pattern
+--------------------------------------------------
+
+When creating a new file, you may choose either to write import statments so that they import the fully-qualified module name:
```

Comment:
"write import statments" → "write module import statements" (to take into account Jim's request).

Reply:
Also, I'd say "when creating a new package".  Each file in a package should endeavor to be consistent with other files in the same package.

Reply:
And it might be nice to mention that an abbreviation chosen should be the same as for any C++ code in the same package (see C++ rule 5-42).